focus-android/app/src/main/res/layout/dialog_add_to_homescreen2.xml

```diff
@@ -82,6 +87,9 @@
             android:layout_width="24dp"
             android:layout_height="24dp"
             android:layout_gravity="center_vertical"
+            android:layout_marginStart="24dp"
+            android:layout_marginTop="26dp"
+            android:layout_marginEnd="32dp"
```

Comment:
I wonder if this value should match marginStart for visual consistency? Appreciate the text is unlikely to be right up against this textview but if it does, should it match the left margin?

Reply:
I've rechecked the designs and that was supposed to be the bottom margin. The margin to ensure space between the icon and text is set by the text view. And it is supposed to be smaller (like in the design) to also align with the title text.